src/main/java/com/linkedin/kmf/KafkaMonitor.java

```diff
@@ -33,12 +42,15 @@
 public class KafkaMonitor {
   private static final Logger LOG = LoggerFactory.getLogger(KafkaMonitor.class);
   public static final String CLASS_NAME_CONFIG = "class.name";
+  private static final String METRIC_GROUP_NAME = "kafka-monitor";
+  private static final String JMX_PREFIX = "kmf";
```

Comment:
That's fine then I don't want to break an existing code.  Does JMX_PREFIX belong in this file if other metrics also use this prefix?